R/commonsurvivalanalysis.R

```diff
@@ -138,6 +191,25 @@
 
   return(as.formula(formula, env = parent.frame(1)))
 }
+.sapGetFormula        <- function(options, modelTerms) {
+
+  predictors    <- .sapGetPredictors(modelTerms)
+  interceptTerm <- options[["includeIntercept"]]
+
+  survival <- .saGetSurv(options)
+
+  if (length(predictors) == 0 && !interceptTerm)
+    stop(gettext("We need at least one predictor, or an intercept to make a formula"))
```

Comment:
If this error message is meant for jasp users (and not for yourself) then complaining about a formula is a bit odd. What about 
```suggestion
    stop(gettext("At least one predictor, or an intercept, is needed to fit the model."))
```